CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## Unreleased
+
+### titiler.docker
+
+- update root certificates at every build (ref: https://github.com/developmentseed/titiler/pull/381)
```

Comment:
As for the other PR, I'm not sure we need to add this in the changelog

Reply:
ok I'll remove it